packs/data/equipment-effects.db/effect-cognitive-mutagen-lesser.json

```diff
@@ -19,28 +19,24 @@
         "rules": [
             {
                 "key": "FlatModifier",
-                "label": "Cognitive Mutagen (Lesser)",
                 "selector": "arcana",
                 "type": "item",
                 "value": 1
             },
             {
                 "key": "FlatModifier",
-                "label": "Cognitive Mutagen (Lesser)",
                 "selector": "crafting",
                 "type": "item",
                 "value": 1
             },
             {
                 "key": "FlatModifier",
-                "label": "Cognitive Mutagen (Lesser)",
                 "selector": "occultism",
                 "type": "item",
                 "value": 1
             },
             {
                 "key": "FlatModifier",
-                "label": "Cognitive Mutagen (Lesser)",
                 "selector": "society",
                 "type": "item",
                 "value": 1
```

Comment:
We'll have to circle back to this to condense these, selectors can be an array now and that will let us make this much cleaner.